index.rst

```diff
@@ -425,56 +425,60 @@ Simulated source spectra
 As mentioned above, we can look in more detail at the spectra of individual sources.
 The simulated images contain roughly 2500 stars ranging from type M to type B, with a distribution that follows local abundances.
 Each star is simulated at high frequency resolution using Kurucz SED profiles and propagated through a model of the LSST g-band filter bandpass, so it is straightforward to compare the measured spectrum of a source to its input spectrum once it is matched.
-To measure the subfilter source spectra, I run a modified version of multiband photometry from the LSST software stack.
+To measure the subfilter source spectra, I run multiband photometry from the LSST software stack with :code:`coaddName=dcr` set.
```

Comment:
I'd prefer to avoid saying “the software stack” where possible, and I think the above could be made a bit more specific.

Could you rephrase to something like “the LSST multiband processing framework, per version ##### of the lsst_distrib codebase”?

Reply:
(I would like to link to the multiband docs but they appear to have gone missing... I am chasing that up! :-))